### PR TITLE
Complete v850 instruction descriptions ##asm

### PR DESCRIPTION
**Pull request by trufae**
copypasta from https://www.renesas.com/us/en/doc/products/mpumcu/doc/v850/r01us0037ej0100_v850e2.pdf

**Your checklist for this pull request**
- [x] I've read the [guidelines for contributing](https://github.com/radareorg/radare2/blob/master/DEVELOPERS.md) to this repository
- [x] I made sure to follow the project's [coding style](https://github.com/radareorg/radare2/blob/master/DEVELOPERS.md#code-style)
- [ ] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the documentation and the [radare2 book](https://github.com/radareorg/radare2book) with the relevant information (if needed)

**Detailed description**

...

**Test plan**

...

**Closing issues**

...

### CODE REVIEW

File: libr/asm/d/v850.sdb.txt [hunk 1]
```diff
@@ -19,27 +22,48 @@ bnl=branch if not carry/less than
...
 in.b=input byte
 in.h=input halfword
 in.w=input word
 jal=jump and link
-jmp=jump register
-jr=jump relative
-ld.b=load byte
+jarl=jump to subroutine and save old pc in given register
+jmp=unconditional jump
+jr=jump to the given routine (not a function, just a regular branch)
```
Comment (wargio) [inline libr/asm/d/v850.sdb.txt:59]: Jump relative not routine

File: libr/asm/d/v850.sdb.txt [hunk 2]
```diff
@@ -19,27 +22,48 @@ bnl=branch if not carry/less than
...
+hsh=halfword swap half word
+hsw=halfword swap word
 in.b=input byte
 in.h=input halfword
 in.w=input word
 jal=jump and link
-jmp=jump register
-jr=jump relative
-ld.b=load byte
+jarl=jump to subroutine and save old pc in given register
```
Comment (wargio) [inline libr/asm/d/v850.sdb.txt:57]: jump and register link

**Review by wargio — CHANGES_REQUESTED** [review]

File: libr/asm/d/v850.sdb.txt [hunk 3]
```diff
@@ -19,27 +22,48 @@ bnl=branch if not carry/less than
...
+jmp=unconditional jump
```
Comment (wargio) [inline libr/asm/d/v850.sdb.txt:58]: jump register

**Review by wargio — CHANGES_REQUESTED** [review]

File: libr/asm/d/v850.sdb.txt (hunk 1 above)
Comment (trufae) [reply libr/asm/d/v850.sdb.txt:59]: Screenshot 2020-10-08 at 19 54 20

File: libr/asm/d/v850.sdb.txt (hunk 3 above)
Comment (trufae) [reply libr/asm/d/v850.sdb.txt:58]: it's register relative, and can also take an imm for disp. but i think the description is pretty clear without so much details. ill change the text with your suggestion

Screenshot 2020-10-08 at 20 24 34

File: libr/asm/d/v850.sdb.txt (hunk 1 above)
Comment (wargio) [reply libr/asm/d/v850.sdb.txt:59]: yes, and it is called `jump relative, not jump to given routine (not a function, just a regular branch)`

File: libr/asm/d/v850.sdb.txt (hunk 3 above)
Comment (wargio) [reply libr/asm/d/v850.sdb.txt:58]: yes, but why so many words? `jump register` is quite clear definition of all that jargon.

**Review by wargio — APPROVED** [review]

**Review by XVilka — APPROVED** [review]

**Pull request merged**